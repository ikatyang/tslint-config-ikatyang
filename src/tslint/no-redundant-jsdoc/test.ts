// good

/**
 * @param a something
 */
function good1(a: string) {

}

// bad

/**
 * @param {string} a something
 */
function bad1(a: string) {

}
