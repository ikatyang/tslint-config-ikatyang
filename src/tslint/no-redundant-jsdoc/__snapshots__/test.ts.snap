// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`lint 1`] = `
// good

/**
 * @param a something
 */
function good1(a: string) {

}

// bad

/**
 * @param {string} a something
          ~~~~~~~~             [Type annotation in JSDoc is redundant in TypeScript code.]
 */
function bad1(a: string) {

}

`;
